packages/calcite-components/src/components/panel/panel.scss

```diff
@@ -3,130 +3,184 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
+ * @prop --calcite-panel-background-color: Specifies the background color of the component.
+ * @prop --calcite-panel-border-color: Specifies the border color of the component.
+ * @prop --calcite-panel-description-color: Specifies the color of the component's description.
+ * @prop --calcite-panel-fab-z-index: Specifies the component fab's z-index.
+ * @prop --calcite-panel-footer-background-color: Specifies the background color of the component's footer.
  * @prop --calcite-panel-footer-padding: Specifies the padding of the component's footer.
+ * @prop --calcite-panel-header-background-color: Specifies the component header's background color.
  * @prop --calcite-panel-header-border-block-end: Specifies the component header's block end border.
+ * @prop --calcite-panel-header-color: Specifies the component header's  color.
+ * @prop --calcite-panel-header-z-index: Specifies the component header's z-index.
  */
```

Comment:
Can we add a `corner-radius` that defaults to "sharp" and a `box-shadow` that defaults to "none" to facilitate theming here? It would facilitate theming these types of layout that use Panel:

![Screenshot 2024-01-30 at 10 58 53 AM](https://github.com/Esri/calcite-design-system/assets/4733155/79425fd0-4a00-4ff4-b7ec-a5ad86accbe8)

Reply:
@macandcheese wouldn't this be out of scope? We're not currently setting any border radius within panel.

Reply:
Yeah, I can open an issue to enable this as a follow up as this should be theme able IMO. We did add these theme able defaults elsewhere in PR for this effort - button, card, meter, some others I think.

Reply:
Yeah i think if we're adding new CSS we should probably have an issue.

It should be a separate feat IMO. cc @jcfranco

Reply:
https://github.com/Esri/calcite-design-system/issues/8675

Reply:
If panel doesn't already have a box-shadow or border-radius and there haven't been any requests from users to add them I would leave them out for now.

Reply:
I have heard requests / designs that call for this, I can share in Teams if needed as they are from unreleased product design. Happy to just do it in the issue I added above as well.

Reply:
I have heard requests / designs that call for this, I can share in Teams if needed as they are from unreleased product design. Happy to just do it in the issue I added above as well which more or less shows a generic version of the request.